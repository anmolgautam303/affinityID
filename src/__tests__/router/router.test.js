import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'

import Router from '../../router/router';

configure({ adapter: new Adapter() });

describe('Router', () => {
    let props = { };
    let router = shallow(<Router {...props}/>);

    it('renders properly', () => {
        expect(Router).toMatchSnapshot();
    });
});