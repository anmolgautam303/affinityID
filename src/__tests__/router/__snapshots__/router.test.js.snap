// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router renders properly 1`] = `[Function]`;
